ansible-advancedlinux/azuredeploy.json

```diff
@@ -0,0 +1,826 @@
+{
+
+	"$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+	"contentVersion": "1.0.0.0",
+
+	"parameters": {
+		"location": {
+			"type": "string",
+			"defaultValue": "West US",
+			"allowedValues": [
+				"West US",
+				"West Europe",
+				"East US",
+				"East Asia",
+				"Southeast Asia"
+			],
+			"metadata": {
+				"Description": "Region where you want to create all the resources"
```

Comment:
I just sent mail. All the templates use this, want to make sure there isn't a case dependency here.